packages/devextreme/js/ui/form.d.ts

```diff
@@ -353,6 +353,13 @@ export default class dxForm extends Widget<dxFormOptions> {
      * @public
      */
     resetValues(): void;
+    /**
+     * @docid
+     * @type_function_param1 editorsData:object
+     * @publicName reset(editorsData)
+     * @public
+     */
+    reset(editorsData?: object): void;
```

Comment:
`object` type should be avoided

```suggestion
    reset(editorsData?: Record<string, any>): void;
```